.circleci/run_acceptance.sh

```diff
@@ -6,5 +6,7 @@ COMMIT_MSG_TITLE=$(git log -n 1 --pretty=format:%s)
 export BROWSERSTACK_TEST_RUN_NAME=$COMMIT_MSG_TITLE
 
 # Browserstack only allows for 2 threads to run at once, so we have to break up the browsers
-npx testcafe "browserstack:firefox,browserstack:chrome" tests/acceptance/acceptancesuite.js
-npx testcafe "browserstack:safari,browserstack:ie@11.0" tests/acceptance/acceptancesuite.js
\ No newline at end of file
+npx testcafe "browserstack:firefox" tests/acceptance/acceptancesuite.js
```

Comment:
oh did we run into clobbering issues?

Reply:
if you're having trouble with browserstack, you can also run tests on browserstack through your local terminal (will probably be much faster)

Reply:
I was just seeing if it was safari or ie which was failing the tests. I'll test it locally and revert this change